### PR TITLE
adding initial pass at hiding team signup UI

### PR DESCRIPTION
**Pull request by jyoo980**
This should hide the "form team" UI before checkpoint 1 is released.

### CODE REVIEW

File: packages/portal/frontend/src/app/custom/CustomStudentView.ts [hunk 1]
```diff
@@ -123,6 +122,17 @@ export class ClassyStudentView extends AbstractStudentView {
         }
     }
 
+    private renderPartnerSelect(): void {
+        // If the result of calling /portal/grades is null, AbstractStudentView sets
+        // the value of this.grades to be an empty array
+        const checkPointOneExists = (transport: GradeTransport): boolean => {
+            return transport.delivId === "c1" && (transport.score === 0 || transport.score === undefined);
+        };
+        if (this.grades.length !== 0 && this.grades.find(checkPointOneExists) !== undefined) {
```
Comment (jyoo980) [inline packages/portal/frontend/src/app/custom/CustomStudentView.ts:131]: `this.grades` is populated with a call to `portal/grades` which then calls `.getGrades`. This is done by the `AbstractStudentView` class.

I'm assuming that when checkpoint 1 is released, there will be a `GradeTransport` associated with it in the call from `.getGrades`

**Comment by braxtonhall** [comment]
Nice this looks perfect! I should actually be able to interactively test this tonight as well. Will update with results.

**Comment by jyoo980** [comment]
Awesome 🎉 

Let me know what happens 👀

**Comment by braxtonhall** [comment]
Dang. This would have been a nice and simple solution but it turns out it doesn't work, so you may need to write a custom endpoint and put it in CustomCourseRoutes. You can model the file like GeneralRoutes, and name the endpoint something related to "partner registration open" instead of "checkpoint 1 open".

Only other request: can you edit the div in the html and give is style display:hidden ? It looks funky popping in and out on the page's initial load.

I did a little poking around and an assignment can be visible to a student, even if their grade hasn't been released, but only if a grade exists (which is kinda silly)

If this seems like a lot let me know and I can work on it soonish as I was able to get through most of the classy stuff, and just have the lecture/collab guide atm.

**Comment by braxtonhall** [comment]
Looks like we'll tackle this one again next term!

**Pull request closed without merging**